packages/xo-server/src/xo-mixins/resource-sets.js

```diff
@@ -124,7 +124,7 @@ export default class {
 
   async computeVmResourcesUsage(vm) {
     return Object.assign(
-      computeVmResourcesUsage(this._xo.getXapi(vm).getObject(vm._xapiId)),
+      computeVmServerResourcesUsage(this._xo.getXapi(vm).getObject(vm._xapiId)),
```

Comment:
Does “server” mean “host” or “pool”?

Reply:
"server" means every resource that isn't stored in XO's DB (ie everything except IP pools).

Reply:
`computeVmXapiResourcesUsage`?